### PR TITLE
Issue templates made.

### PR DESCRIPTION
**Pull request by singh-anushka**
## Describe your changes
Issue templates added.
Added a .github folder where yml files for issue templates are added.
Fixes [#208](https://github.com/amupedia2021/Project-Amupedia/issues/208)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Starred the repository.
- [ ] Added My Name to the Contributor's List.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.

## Additional Information (Optional)
Any additional information that you want to give us

### CODE REVIEW

**Review by anmode — CHANGES_REQUESTED** [review]
Looks cool!!

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      value: "Description"
+    validations:
+      required: true
+      
+      
+  - type: textarea
+    id: instructions
+    attributes:
+      label: Provide breif description to get to the expect behaviour.
+      description: Enter the breif description on how you plan to find the bug's solution
```
Comment (anmode) [inline .github/ISSUE_TEMPLATE/bug_report.yml:39]: grammatical mistakes. Please fix them

Comment (singh-anushka) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Done!

**Comment by anmode** [comment]
cool! Congratss

**Comment by singh-anushka** [comment]
Kindly apply gssoc23 label too.

**Pull request merged**